### PR TITLE
[cli] Wrap description in help output

### PR DESCRIPTION
**Pull request by trek**
If we wanted to avoid pulling in another dependency (`word-wrap`), this is a too-clever-by-half way to get word wrapping on the `--help` description text.

Before:
CleanShot 2023-08-18 at 10 11 14@2x

After:
CleanShot 2023-08-18 at 10 08 52@2x

### CODE REVIEW

File: packages/cli/src/commands/help.ts [hunk 1]
```diff
@@ -277,9 +282,25 @@ export function buildCommandExampleLines(command: Command) {
...
+      'padding-left': INDENT.length,
+    },
+  });
+  const table = new Table(_tableOptions);
+  table.push([
+    {
+      content: command.description,
+      wordWrap: true,
+    } as _CellOptions,
+  ]);
```
Comment (EndangeredMassa) [inline packages/cli/src/commands/help.ts]: **suggestion (soft-blocking):** extract function

Can we extract this into a named function to make it super clear that we're using this table module purely for word wrapping?‏

**Review by cb1kenobi — DISMISSED** [review]
LGTM

File: packages/cli/src/commands/help.ts (hunk 1 above)
Comment (EndangeredMassa) [reply packages/cli/src/commands/help.ts]: **alternate suggestion (non-blocking):** use `word-wrap`

I don't think the module [word-wrap](https://github.com/jonschlinkert/word-wrap) is [too big](https://packagephobia.com/result?p=word-wrap) to bring in. I'd be fine with that.

**Review by EndangeredMassa — DISMISSED** [review]
I don't love using a table module for this, but if we don't want another dependency, it does work.

File: packages/cli/src/commands/help.ts (hunk 1 above)
Comment (cb1kenobi) [reply packages/cli/src/commands/help.ts]: If we use `word-wrap` will the wrapped description continue to be indented on subsequent lines?

Comment (trek) [reply packages/cli/src/commands/help.ts]: @cb1kenobi yessir.

**Review by Ethan-Arrowood — DISMISSED** [review]

File: packages/cli/src/commands/help.ts (hunk 1 above)
Comment (trek) [reply packages/cli/src/commands/help.ts]: Tested out `word-wrap` and hilariously it does a poorer job unless you manually add a bit of right padding. It seems to struggle with word boundaries:

CleanShot 2023-08-18 at 13 34 56@2x

I do like Sean's suggestion of extracting into a function.

**Review by cb1kenobi — APPROVED** [review]
Tested and looks good!

**Pull request merged**